### ISSUE TITLE
Problem on boot

### ISSUE THREAD

**Issue by cadupalmieri** [defencedog/orangepi3b_v2.1#4]
Hi, i am doing the process complete as you write but the final step, removing the sd card and boot in nvme this text appers and the system dont init

Loading, please wait...

Starting systend-udeud version 252.22-1*deb12u1

done.

Begin: Loading essential drivers ... done.

Begin: Running /scripts/init-premount ... (initranfs)

**Comment by defencedog** [comment]
Which process you are referring to?

**Comment by cadupalmieri** [comment]
> Which process you are referring to?

Cloning sd card to nvme on orangepi3b

**Comment by defencedog** [comment]
Have you done x2 steps mentioned in _Update bootloader_ ...don not attach any other USB before doing this

**Comment by cadupalmieri** [comment]
> Have you done x2 steps mentioned in _Update bootloader_ ...don not attach any other USB before doing this

Yes! And I try to check with lsblk, the mountpoints on Nvme stay blank , it was supposed to be like this example.

nvme0n1     259:0    0 238.5G  0 disk 
├─nvme0n1p1 259:1    0     1G  0 part /boot
└─nvme0n1p2 259:2    0   235G  0 part /var/log.hdd

but stays 
nvme0n1     259:0    0 238.5G  0 disk

**Comment by defencedog** [comment]
I am at lost here I may not be able to help further. However, you are welcome to share solution to this if you find. I do recommend starting again with an sdcard & repeating steps
